testing/kfctl/conftest.py

```diff
@@ -24,6 +24,10 @@ def pytest_addoption(parser):
   parser.addoption(
       "--use_istio", action="store", default="False",
       help="Use istio.")
+  
+  parser.addoption(
+      "--package_manager", action="store", default="False",
```

Comment:
Nit: Give this flag a more descriptive name; e.g. use_kustomize

ksonnet and kustomize are both package managers.

Reply:
Okay, will do

Reply:
But since we're using the same flag for kfctl is better to keep it the same here?